### PR TITLE
Update licensing topic with new section

### PR DESCRIPTION
**Pull request by zdrawku**
Closes #2060

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: en/components/general/ignite-ui-licensing.md [hunk 1]
```diff
@@ -152,24 +152,44 @@ The following information is on how to setup authentication to our private npm r
...
+- script: npm install
+  displayName: 'Install dependencies'
+  env:
+    AZURE_PIPELINES: "true"
+
+- script: npm run lint
+  displayName: 'Run lint'
+
+- script: npm run build
+  displayName: 'Generate live editing and build samples'
```
Comment (damyanpetev) [inline en/components/general/ignite-ui-licensing.md:178]: Huh?
What are those extra stuff? Generate live editing, Lint and the `AZURE_PIPELINES` all seem out of scope

File: en/components/general/ignite-ui-licensing.md [hunk 2]
```diff
@@ -152,24 +152,44 @@ The following information is on how to setup authentication to our private npm r
...
 
-<img class="responsive-img" style="-webkit-box-shadow: 8px 9px 9px 5px #ccc; -moz-box-shadow: 8px 9px 9px 5px #ccc; box-shadow: 8px 9px 9px 5px #ccc; min-width: calc(100% - 450px)" 
-  src="../../images/general/azure-ci-pipelines-ci-yml-3.jpg"
-  data-src="../../images/general/azure-ci-pipelines-ci-yml-3.jpg"
-  alt="Azure Pipelines CI yml update"
-  title="Azure Pipelines CI yml update" />
+Now we need to add variables for the *npm registry*, *scope* and *token*. There are two ways to do so:
 
-Add **npm registry** and **token** variables.
+ #### [Define Variable Group](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml) from the Library page under Pipelines.
```
Comment (damyanpetev) [inline en/components/general/ignite-ui-licensing.md:184]: Oh boy, using small headers as emphasis and for entire sentences with links. You can bold.. or put a single line of text under.

File: en/components/general/ignite-ui-licensing.md [hunk 3]
```diff
@@ -197,3 +217,16 @@ The best way to define an environment variable depends on what type of informati
 
 * encrypt it and add it [to your .travis.yml](https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml)
 * add it to your [Repository Settings](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings)
+
+### GitHub Actions Configuration
+Azure Pipelines and GitHub Actions both allow you to [create workflows](https://help.github.com/en/actions/migrating-to-github-actions/migrating-from-azure-pipelines-to-github-actions#migrating-script-steps) that automatically build, test, publish, release, and deploy code.
```
Comment (damyanpetev) [inline en/components/general/ignite-ui-licensing.md:222]: What's Azure pipelines have to do with all this? And the migration link - no getting started link available?

File: en/components/general/ignite-ui-licensing.md [hunk 4]
```diff
@@ -197,3 +217,16 @@ The best way to define an environment variable depends on what type of informati
...
+
+Add the following scripts before the `npm i(ci)` script:
```
Comment (damyanpetev) [inline en/components/general/ignite-ui-licensing.md:224]: Do say in this is in the wokflow YAML file (whatever the name is)

File: en/components/general/ignite-ui-licensing.md [hunk 5]
```diff
@@ -152,24 +152,44 @@ The following information is on how to setup authentication to our private npm r
...
-```
+pool:
+  vmImage: 'Ubuntu 16.04'
 
-```cmd
-npm config set always-auth true --scope:@infragistics
-```
+steps:
+
+- script: echo "@infragistics:registry=$(npmRegistry)" >> ~/.npmrc
```
Comment (kdinev) [inline en/components/general/ignite-ui-licensing.md:160]: You can keep the old `npm config set` scripts, or would you like to streamline all guidance to `echo` into the `.npmrc` file? Also you define a variable group below, but you never set it in the script:

```
variables:
- group: Common-variables
```

**Comment by zdrawku** [comment]
@damyanpetev, the snippets will be updated tomorrow. we need to perform additional testing based on the snippets
